src/test/java/de/mediathekview/mserver/crawler/zdf/json/ZdfDayPageDeserializerTest.java

```diff
@@ -0,0 +1,124 @@
+package de.mediathekview.mserver.crawler.zdf.json;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.notNullValue;
+
+import com.google.gson.JsonObject;
+import de.mediathekview.mserver.crawler.zdf.ZdfEntryDto;
+import de.mediathekview.mserver.testhelper.JsonFileReader;
+import de.mediathekview.mserver.testhelper.WireMockTestBase;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Optional;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+
+@RunWith(Parameterized.class)
+public class ZdfDayPageDeserializerTest {
+
+  @Parameters
+  public static Collection<Object[]> data() {
+    return Arrays.asList(new Object[][]{
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this array creation and simply pass the elements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3878)